apps/comments-ui/src/actions.ts

```diff
@@ -449,7 +466,9 @@ export const Actions = {
     loadMoreReplies,
     updateMember,
     setOrder,
-    openCommentForm
+    openCommentForm,
+    setHighlightedRepliedToComment,
+    removeHighlightedRepliedToComment
```

Comment:
Can we change the action names to hide the internal implementation a bit? `dispatchAction('highlightComment', commentId)` reads a lot nicer 😄